tests/Doctrine/Tests/Common/DataFixtures/TestEntity/ReservedKeyword.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Doctrine\Tests\Common\DataFixtures\TestEntity;
+
+/**
+ * @Entity
+ * @Table("SELECT")
```

Comment:
Should be quoted here, as per https://doctrine-orm.readthedocs.io/projects/doctrine-orm/en/latest/reference/limitations-and-known-issues.html#identifier-quoting-and-legacy-databases